ext/microprofile/mp-rest-client/pom.xml

```diff
@@ -121,6 +121,7 @@
                             ${cdi.osgi.version},
                             jakarta.decorator.*;version="[3.0,5)",
                             org.eclipse.microprofile.config.*;version="!",
+                            org.eclipse.microprofile.rest.client;version="[2.0,3)",
```

Comment:
Thanks for telling us. But the version used to be 3, now it is 4, so the range better be [3.0, 5).

Reply:
> Maybe version="!" looks even better.

Is that the equivalent of simply removing version validation?

Ideally we'd need to do that for all packages (since they're all versioned separately) to prevent only validating _some_ of them